dev-utils/storybook/stories/lit-components/button.stories.js

```diff
@@ -0,0 +1,102 @@
+import { html } from "lit-element";
+import "../../../../feature-utils/poly-look/src/lit-components/buttons";
+import { themeConfiguration } from "./themeConfiguration";
+
+export default {
+    title: "Visuals/Atoms/Button",
+    component: "poly-button",
+    argTypes: {
+        type: { control: "text" },
+        size: { control: "text" },
+        disabled: { control: "boolean" },
+        onPolyButtonClicked: {
+            action: "polyButtonClicked",
+        },
+    },
+};
+
+function Template({
+    type = "dark",
+    size = "medium",
+    disabled,
+    onPolyButtonClicked,
+}) {
+    return html`
+        ${themeConfiguration()}
+        ${type === "light"
+            ? html`<style>
+                  body {
+                      background-color: #0f1938;
+                  }
+              </style>`
+            : ""}
+        <style>
+            .button-area {
+                width: 80%;
+                padding-left: 32px;
+                padding-right: 32px;
+                padding-bottom: 32px;
+                height: 200px;
+            }
+        </style>
+        <div class="button-area">
+            <poly-button
+                .type=${type}
+                .size=${size}
+                .disabled=${disabled}
+                @poly-button-clicked=${onPolyButtonClicked}
+                >Example</poly-button
+            >
+        </div>
+    `;
+}
+
+export const Regular = Template.bind({});
```

Comment:
Again, is it possible to avoid boilerplate by using a loop for this? It's just the same code repeated with different values for size.